patroni/postgresql/bootstrap.py

```diff
@@ -400,9 +400,9 @@ def post_bootstrap(self, config: Dict[str, Any], task: CriticalTask) -> Optional
 END;$$""".format(f, quote_ident(rewind['username'], postgresql.connection()))
                         postgresql.query(sql)
 
-                for name, value in (config.get('users') or {}).items():
-                    if all(name != a.get('username') for a in (superuser, replication, rewind)):
-                        self.create_or_update_role(name, value.get('password'), value.get('options', []))
+                if config.get('users'):
+                    logger.error('user creation is not be supported after v4.0.0')
```

Comment:
```suggestion
                    logger.error('User creation via "bootstrap.users" is not supported since v4.0.0')
                    logger.info('Please use "boostrap.post_bootstrap" script to create users.')
```